examples/input.rs

```diff
@@ -0,0 +1,62 @@
+extern crate tetra;
+
+use tetra::glm::Vec2;
+use tetra::graphics::{self, Color};
+use tetra::graphics::text::{Text, Font};
+use tetra::input::{self, Key};
+use tetra::{Context, ContextBuilder, State};
+
+struct GameState {
+    input: String,
+    text: Text,
+}
+
+impl GameState {
```

Comment:
You can get rid of this impl block now, don't think it's being used anywhere else.

Reply:
Derp, that's what I get for committing without testing